### ISSUE TITLE
Crafting Terminal not working exactly as it should (1.11.2)

### ISSUE THREAD

**Issue by DatrixTHLK** [zerofall/EZStorage2#59]
Hi, 

For some reason I'm only able to craft at most 1 item out of a whole stack of items. If I wanted to make a stack of iron blocks out of iron ingots, I would just shift and click on the iron block on the output. 1.10.2 did that. 1.11.2 isn't doing that. The crafting box is making the items disappear from the crafting terminal after I create just one of those items or in effect spitting them back into the storage core.

Minecraft 1.11.2
forge 13.20.1.2588
ezstorage 2.4.1

**Comment by sblectric** [comment]
yeah the crafting interface has a few weird quirks to be ironed out eventually

**Comment by DatrixTHLK** [comment]
Thanks for updating the mod though. :)

**Comment by Darkosto** [comment]
Hello!

I would love to use this mod in my next modpack and was hoping you had an update on whether or not this issue was going to be looked at in the near future? 

Thank you :D

**Comment by oceanog** [comment]
I have the same exact problem in 1.12.2. Please fix!

**Comment by DatrixTHLK** [comment]
Mod is abandoned. No point. Disappointing since I found this storage mod the easiest to use out of all the highly technical storage mods I've played such as AE2.

**Comment by ShadowSharkWolf** [comment]
Too Bad... I come back to this every once in a while just see if anything has been done for this and I know this is necroing a dead post, but man I wish I could play this in 1.18.2 and 1.19.x.  If not, oh well...
